_posts/2022-03-25-zero-cost-proxies.md

```diff
@@ -0,0 +1,476 @@
+---
+layout: post
+title: A Deeper Look at Zero-Cost Proxies for Lightweight NAS  
+authors: Anonymous
+tags: [deep learning, neural architecture search, zero-cost proxies]  # This should be the relevant areas related to your blog post
+---
+
+<img src="{{ site.url }}/public/images/2021-12-01-zero-cost-proxies/MainFigure.png"
+     alt="Markdown Monster icon"
+     style="float: left; margin-right: 10px;" />
+
+Imagine you have a brand new dataset, and you are trying to find a neural network that achieves high validation accuracy on this dataset. You choose a neural network, but after 3 hours of training, you find that the validation accuracy is only 85%. After more choices of neural networks --- and many GPU-hours --- you finally find one that has an accuracy of 93%. Is there an even better neural network? And can this whole process become faster?
```

Comment:
I'd propose to swap the order of these two things, i.e.:

```
Imagine you have a brand new dataset, and you are trying to find a neural network that achieves high validation 
accuracy on this dataset. You choose a neural network, but after 3 hours of training, you find that the validation 
accuracy is only 85%. After more choices of neural networks --- and many GPU-hours --- you finally find one that has 
an accuracy of 93%. Is there an even better neural network? And can this whole process become faster?


<img src="{{ site.url }}/public/images/2022-03-25-zero-cost-proxies/MainFigure.png"
     alt="Markdown Monster icon"
     style="float: left; margin-right: 10px;" />
```

This will make the post excerpt in the `blog/` endpoint render a summary rather than the image

Reply:
Done [in this commit](https://github.com/iclr-blog-track/iclr-blog-track.github.io/pull/34/commits/c5316c419f972695bf4d4671fdc5a685b195f560)